backstop/tests/wizard.html

```diff
@@ -33,175 +33,247 @@
     <theme-button></theme-button>
     <h1>Wizard</h1>
     <hr />
-    <h2 id="default">Default</h2>
 
+    <h2>Default</h2>
     <section id="demo-default">
-      <div class="iui-wizards">
-        <div class="iui-wizards-wrapper">
-          <span class="iui-wizards-step iui-wizards-step-completed iui-clickable">
-            <span class="iui-wizards-step-title">First step</span>
-            <span class="iui-wizards-step-track iui-wizards-step-track-after"></span>
-            <span
-              class="iui-wizards-step-circle"
-              tabindex="0"
-            >1</span>
-          </span>
 
-          <span class="iui-wizards-step-track iui-wizards-step-track-main iui-wizards-step-track-filled"></span>
-
-          <span class="iui-wizards-step iui-wizards-step-completed iui-clickable">
-            <span class="iui-wizards-step-track iui-wizards-step-track-before"></span>
-            <span class="iui-wizards-step-title">Completed step</span>
-            <span class="iui-wizards-step-track iui-wizards-step-track-after"></span>
-            <span
-              class="iui-wizards-step-circle"
-              tabindex="0"
-            >
-              <span>2</span>
-            </span>
+      <div class="iui-wizard">
+        <nav
```

Comment:
Can we really use `<nav>` here? It is meant to be used for navigation, also W3 is using `<ol>` in their examples https://www.w3.org/WAI/tutorials/forms/multi-page/#using-the-html5-progress-element

Reply:
Swapped from `<nav>` & `<span>` to `<ol>` & `<li>` in the next push.